mars/oscar/backends/ray/driver.py

```diff
@@ -50,7 +50,9 @@ def setup_cluster(cls, address_to_resources: Dict[str, Dict[str, Number]]):
                 num_cpus=0,  # main pool doesn't do horse work, mark it use no cpu.
                 name=address, placement_group=pg, placement_group_bundle_index=index).remote()
             n_process = bundle_spec["CPU"]
-            ray.get(actor_handle.start.remote(address, n_process))
+            # TODO move this to a function
```

Comment:
Maybe we can move the start actor pool logic to `RayActorBackend.create_actor_pool`? Currently, the `RayActorBackend.create_actor_pool` is empty.